examples/create_services.sh

```diff
@@ -172,4 +172,14 @@ else
   rm tmp-notify-variable-service.sh
 fi
 
+if cf service os-places-variable-service >/dev/null 2>/dev/null; then
+    echo "os-places-service already exists"
+else
+  echo "Setting up os-places variable service to provide notify api key to apps"
+  echo "cf create-user-provided-service os-places-variable-service -p '{\"OS_PLACES_API_KEY\": \"${OS_PLACES_API_KEY}\"}'"
+  echo "cf create-user-provided-service os-places-variable-service -p '{\"OS_PLACES_API_KEY\": \"${OS_PLACES_API_KEY}\"}'" > tmp-os-places-variable-service.sh
```

Comment:
We'll need the OS places url as well as the key. Shall we put them in the same service?

Reply:
Updated to include uri.